sql/core/src/main/java/org/apache/spark/sql/execution/datasources/parquet/VectorizedRleValuesReader.java

```diff
@@ -928,11 +956,35 @@ private boolean readNextGroup() {
           }
           currentBufferIdx = 0;
           int valueIndex = 0;
-          while (valueIndex < this.currentCount) {
-            // values are bit packed 8 at a time, so reading bitWidth will always work
-            ByteBuffer buffer = in.slice(bitWidth);
-            this.packer.unpack8Values(buffer, buffer.position(), this.currentBuffer, valueIndex);
-            valueIndex += 8;
+          if (vector512Support && SQLConf.get().parquetVector512Read()) {
```

Comment:
It seems that there is currently no corresponding UTs, and if not all GA machines can be used to check this part, how can we continue to ensure its robustness?